src/platform/mhv/api/mocks/secure-messaging/recipients/index.js

```diff
@@ -132,9 +132,325 @@ const recipients = {
         relationshipType: 'PATIENT',
       },
     },
+    {
+      id: '2994573',
+      type: 'all_triage_teams',
+      attributes: {
+        triageTeamId: 2994573,
+        name: 'GLEN_BURNIE_PRIMARY_CARE_SMITH_ALLEN',
+        stationNumber: '512',
+        blockedStatus: false,
+        preferredTeam: true,
+        relationshipType: 'PATIENT',
+        leadProviderName: null,
+        locationName: 'Baltimore VA Medical Center',
+        teamName: null,
+        suggestedNameDisplay:
+          'Glen Burnie VA  Clinic | Primary Care | Team 3 - Allen Smith',
```

Comment:
Is there an extra space between VA and Clinic?

Reply:
probably just copy pasting from Figma, but I can correct this